requirements/edx/base.txt

```diff
@@ -35,11 +35,14 @@ django-statici18n==1.4.0
 django-storages==1.4.1
 django-method-override==0.1.0
 django-user-tasks==0.1.4
+# Why a fork? See: https://openedx.atlassian.net/browse/PLAT-1581
 git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
 django==1.8.18
 django-waffle==0.12.0
 djangorestframework-jwt==1.11.0
-djangorestframework-oauth==1.1.0
+# Why a fork? To add Django 1.11 compatibility to the abandoned repo.
+# This dependency will be removed by this work: https://openedx.atlassian.net/browse/PLAT-1660
+git+https://github.com/edx/django-rest-framework-oauth.git@0a43e8525f1e3048efe4bc70c03de308a277197c#egg=djangorestframework-oauth==1.1.1
```

Comment:
Pure sloppiness - along with an attempt at self-deception that I'm actually adding a dependency on a fork. I'm going to stop fooling myself and fix this.